node/src/accountant/mod.rs

```diff
@@ -1791,7 +1803,7 @@ mod tests {
         );
         assert!(
             before <= captured_now && captured_now <= after,
-            "captured timestamp should have been between {:?} and {:?} but was {:?}",
+            "captured should be between {:?} and {:?} but was {:?}",
```

Comment:
Either you should replace the word `captured` with `it` or you should introduce `timestamp` after `captured` to make it clear.